lib/timer.py

```diff
@@ -120,6 +123,11 @@ def run_interval_seconds(self, cfg: UAConfig) -> int:
     update_contract_info,
     UPDATE_CONTRACT_INFO_INTERVAL,
 )
+validate_release_series_job = TimedJob(
+    "validate_release_series",
+    validate_release_series,
+    UPDATE_CONTRACT_INFO_INTERVAL,
```

Comment:
I know we want the same value, but we should give this it's own variable to hold the INTERVAL value.